frontend/components/ifixit-image/ifixit-image.tsx

```diff
@@ -1,32 +1,34 @@
 import Image, { ImageProps, ImageLoader, ImageLoaderProps } from 'next/image';
 
-interface SizeMap {
-   [index: string]: number;
+interface SizeMapEntry {
+   width: number;
+   name: string;
 }
+type SizeMap = Array<SizeMapEntry>;
 
-const guideImageSizeMap: SizeMap = {
-   mini: 56,
-   thumbnail: 96,
-   '140x105': 140,
-   '200x150': 200,
-   standard: 300,
-   '440x330': 440,
-   medium: 592,
-   large: 800,
-   huge: 1600,
-};
+const guideImageSizeMap: SizeMap = [
```

Comment:
A comment indicating that these need to be sorted, or using a name like `sortedGuideImageSizeMap`, would make me feel a little better that no one will accidentally unsort these.